wrapanapi/systems/rhevm.py

```diff
@@ -855,23 +866,33 @@ def list_cluster(self, **kwargs):
     def get_disks(self, name):
         return self.api.system_service().disks_service().list(search=name)
 
-    def list_disks(self, status=None, **kwargs):
+    def list_disks(self, status=None, sparse=None, disk_id=None, **kwargs):
```

Comment:
Starting to get a little squirelly here, consider creating a disk entity, and returning a list of disk entities instead. Then the user can filter those on sparse, or select ids/names as they want.